iec_api/iec_client.py

```diff
@@ -524,6 +524,9 @@ async def get_power_size(self, connection: Optional[str] = None) -> float:
 
             connection = device_details.counter_devices[0].connection_size.representative_connection_size
 
+        if 'X' not in connection: # Solve cases where the connection size is "25"
```

Comment:
FWIW, I think this was my issue as well; I have single phase, 40 amps, and I was getting a KeyError on `40`

Reply:
Yep.